src/main/scala/devices/spi/TLSPI.scala

```diff
@@ -120,7 +121,13 @@ class SPITopModule(c: SPIParamsBase, outer: TLSPIBase)
       RegField.r(1, ip.txwm,
       RegFieldDesc("txwm_ip","Transmit watermark interupt pending", volatile=true)),
       RegField.r(1, ip.rxwm,
-      RegFieldDesc("rxwm_ip","Receive watermark interupt pending", volatile=true)))))
+      RegFieldDesc("rxwm_ip","Receive watermark interupt pending", volatile=true)))),
+
+    SPICRs.extradel ->  RegFieldGroup("extradel",Some("delay from the sck edge"),Seq(
+      RegField(c.divisorBits, ctrl.extradel.coarse,
+               RegFieldDesc("extradel_coarse","Coarse grain sample delay", reset=Some(0))),
+      RegField(5, ctrl.extradel.fine,
```

Comment:
since c.DivisorBits is a variable, this field could move around in this register. Is that what you want? I would have guessed you would want it at some fixed offset